app/ui/modal-dialog-custom.d.ts

```diff
@@ -0,0 +1,30 @@
+export function alert(title: any, text: any): {
```

Comment:
Do we need these declaration files on github repo?

Reply:
Will exclude them right away. They might have been generated when I was generating types for the pluginManager-comoponent.js file.